qutebrowser/mainwindow/mainwindow.py

```diff
@@ -344,6 +344,12 @@ def closeEvent(self, e):
             quit_texts.append("{} {} running.".format(
                 tab_count,
                 "download is" if tab_count == 1 else "downloads are"))
+        # Check if any tabs have filled forms and skip
+        if 'forms' in confirm_quit:
+            for tab in self._tabbed_browser.widgets():
+                if not self._tabbed_browser.check_forms_can_be_destroyed(tab):
+                    e.ignore()
+                return
```

Comment:
My reason for only checking the first tab was that it didn't seem to be helpful to display multiple questions about quitting.  Currently the browser doesn't jump to your tab with filled form data.  So having the question appear multiple times isn't very helpful to the user.

If we had it jump to the filled tab then that approach might make sense.